### PR TITLE
allow handpose to execute on react native and tensorflow backends

### PR DESCRIPTION
**Pull request by tafsiri**
Should address this issue https://github.com/tensorflow/tfjs/issues/2906

What do you think @annxingyuan. Also if this looks good to you are version bumps done via scripts or directly in the PR making the change.

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-models/446)

### CODE REVIEW

**Comment by tafsiri** [comment]
i also had trouble running tests locally, i tried to add tfjs-backend-webgl from npm but it still errored out.

**Comment by tafsiri** [comment]
@annxingyuan please re-review, i disabled webgpu for now and enabled tests in CI.

File: cloudbuild.yml [hunk 1]
```diff
@@ -77,6 +77,13 @@ steps:
   args: ['./scripts/run-build.sh', 'universal-sentence-encoder']
   waitFor: ['diff']
 
+# Handpose
+- name: 'gcr.io/cloud-builders/gcloud'
+  entrypoint: 'bash'
+  id: 'handpose'
+  args: ['./scripts/run-build.sh', 'handpose']
+  waitFor: ['diff']
+
```
Comment (annxingyuan) [inline cloudbuild.yml:86]: Could we add facemesh here as well, while you're touching this file?

Comment (tafsiri) [reply cloudbuild.yml:86]: done

**Pull request merged**